test/NuGet.Core.Tests/NuGet.Commands.Test/PackCommandRunnerTests.cs

```diff
@@ -48,6 +48,71 @@ public void RunPackageBuild_WithDefaultExcludes_ExcludesDefaultExcludes()
             }
         }
 
+        [Fact]
+        public void RunPackageBuild_WithInvalidNuspec_Throws()
+        {
+            using (var test = DefaultExclusionsTest.Create(validNuspec: false))
+            {
+                var args = new PackArgs()
+                {
+                    CurrentDirectory = test.CurrentDirectory.FullName,
+                    Exclude = Enumerable.Empty<string>(),
+                    Logger = NullLogger.Instance,
+                    Path = test.NuspecFile.FullName,
+                };
+                var runner = new PackCommandRunner(args, createProjectFactory: null);
+
+                Assert.Throws<Exception>(() => runner.RunPackageBuild());
+
+                var nupkgPath = Path.Combine(test.CurrentDirectory.FullName, $"DefaultExclusions.1.0.0.nupkg");
+                Assert.False(File.Exists(nupkgPath), "The output .nupkg should not exist when pack fails.");
+            }
+        }
+
+        [Fact]
+        public void RunPackageBuild_WithValidNuspec_WithVersionAsArgument_PackSucceeds()
```

Comment:
```suggestion
        public void RunPackageBuild_WithVersionAsArgument_NuspecVersionOverriddenByVersionArgument()
```

if the "scenario" part of the name doesn't mention inputs being invalid, I think it's safe to assume that the inputs are valid, so adding it to the name feels redundant.

Also, result "Pack Succeeds" is just saying it doesn't throw an exception (or some other kind of failure).  But the test is validating more than just no exception is thrown.